### PR TITLE
Handle `ResourceDeleted` websocket notification for actions on patient flow page

### PR DESCRIPTION
**Pull request by nmajor25**
Shortcut Story ID: [sc-56332]

When this happens, we want to remove the action from the list via `destroy()` and close the action sidebar if it's open.

We added some code to the base model `destroy()` method to stop the `DELETE` api request in this scenario.

### CODE REVIEW

File: src/js/apps/patients/sidebar/action-sidebar_app.js [hunk 1]
```diff
@@ -44,6 +44,7 @@ export default App.extend(extend({
     const flow = this.action.getFlow();
     if (flow) this.listenTo(flow, 'change:_state', this.setAccess);
     this.listenTo(action, 'change:_owner', this.onChangeOwner);
+    this.listenTo(action, 'destroy', this.stop);
```
Comment (paulfalgout) [inline src/js/apps/patients/sidebar/action-sidebar_app.js]: I think the two action listenTos can be made into one with {}

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/apps/patients/sidebar/action-sidebar_app.js (hunk 1 above)
Comment (nmajor25) [reply src/js/apps/patients/sidebar/action-sidebar_app.js]: Updated, much cleaner 👍

**Review by paulfalgout — APPROVED** [review]

**Comment by paulfalgout** [comment]
needs a rebase @nmajor25

**Comment by nmajor25** [comment]
@paulfalgout rebased, should be good now.

**Pull request merged**